### PR TITLE
add Twilio and REST-client

### PR DESCRIPTION
**Pull request by ghost**
Included the example text from Twilio and REST-client. Should these gems still be part of the primary app file or should they be moved to a gemfile?

### CODE REVIEW

**Comment by matugm** [comment]
Can we have the examples in a separate file? Maybe make it into a markdown file for easier reading.

**Comment by ghost** [comment]
Created app. Put gems in Gemfile.

**Comment by matugm** [comment]
What do you think about separating the examples into two code blocks, one for twillio and one for rest-client? It might be easier to read that way.

Another alternative for the documentation data could be using the Wiki feature from github:

image

Also I don't think `game.rb` is going to need any of the requires because it will contain only the game logic and nothing else.

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# A sample Gemfile
+source "https://rubygems.org"
+
+# gem "rails"
```
Comment (matugm) [inline Gemfile]: We don't need rails, so you can delete this line.

**Comment by matugm** [comment]
We have decided to use the Wiki for documentation on the last Ruby Monday.

Since this file (`gem_examples.md`) is just documentation, the Wiki would be a better place for it.

**Comment by luminousbeam** [comment]
We will be adding the gems once we finish with the game logic.

**Pull request closed without merging**